db/migrate/20210223152748_add_operator_to_relative_date_filters.rb

```diff
@@ -0,0 +1,43 @@
+class AddOperatorToRelativeDateFilters < ActiveRecord::Migration[6.1]
```

Comment:
Should this actually just be a script we run on release day rather than a migration? Nothing is changing about the schema. Thoughts @timwongj @holmesie?

Reply:
It could be a script that we run on release day rather than a migration. I just created it as a migration because I thought it would be easier to revert things in case anything went wrong

Reply:
Discussed with Adam, we would like this to be a script we run rather than a migration. You can share the script file with us in a DM for tracking.

Reply:
migration has been removed